ldap-scim-bridge.cabal

```diff
@@ -25,28 +25,28 @@ source-repository head
   location:            https://github.com/wireapp/ldap-scim-bridge.git
 
 common common-options
-  build-depends:       base >=4.13 && <4.15
-                     , relude
-                     , hscim >=0.3.6 && <0.4
-                     , ldap-client >=0.4.1 && <0.5
-                     , network >=3.1.1.1 && <3.2
-                     , text >=1.2.4.1 && <1.3
-                     , yaml >=0.11.5.0 && <0.12
-                     , aeson >=1.4.7.1 && <1.5
-                     , aeson-pretty >=0.8.8 && <0.9
-                     , containers >=0.6.2.1 && <0.7
-                     , bytestring >=0.10.12 && <0.11
-                     , email-validate >=2.3.2.13 && <2.4
-                     , string-conversions >=0.4.0.1 && <0.5
-                     , servant-client >=0.18.3 && <0.19
-                     , unordered-containers >= 0.2.14.0 && <0.3
-                     , servant-client-core >=0.18.3 && <0.19
-                     , servant >=0.18.3 && <0.19
-                     , http-types >=0.12.3 && <0.13
-                     , string-conversions >=0.4.0.1 && <0.5
-                     , http-client >=0.7.8 && <0.8
-                     , http-client-tls >=0.3.5 && <0.4
-                     , tinylog >=0.15 && <0.16
+  build-depends:       base
+                     , aeson >= 2.1.2 && < 2.2
+                     , aeson-pretty >= 0.8.10 && < 0.9
+                     , base >= 4.17.2 && < 4.18
+                     , bytestring >= 0.11.5 && < 0.12
+                     , containers >= 0.6.7 && < 0.7
+                     , email-validate >= 2.3.2 && < 2.4
+                     , hscim >= 0.4.0 && < 0.5
```

Comment:
Please make the lower bound 0.4.0.2, I forgot to (but will now) deprecate the other 4.* releases.